cmd/common.go

```diff
@@ -2,20 +2,37 @@ package cmd
 
 import (
 	"fmt"
+
 	"github.com/appuio/seiso/cfg"
 	"github.com/appuio/seiso/pkg/git"
+	"github.com/appuio/seiso/pkg/kubernetes"
 	"github.com/appuio/seiso/pkg/openshift"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
-func DeleteImages(imageTags []string, imageName string, namespace string) {
+// DeleteImages deletes a list of image tags
+func DeleteImages(imageTags []string, imageName string, namespace string, force bool) {
+	if !force {
+		log.Warn("Force mode not enabled, nothing will be deleted")
+	}
 	for _, inactiveTag := range imageTags {
-		err := openshift.DeleteImageStreamTag(namespace, openshift.BuildImageStreamTagName(imageName, inactiveTag))
-		if err == nil {
-			log.WithField("imageTag", inactiveTag).Info("Deleted image tag")
+		logFields := log.Fields{
+			"namespace": namespace,
+			"image":     imageName,
+			"imageTag":  inactiveTag,
+		}
+		if force {
+			if err := openshift.DeleteImageStreamTag(namespace, openshift.BuildImageStreamTagName(imageName, inactiveTag)); err != nil {
+				log.WithFields(logFields).
+					Info("Deleted image tag")
+			} else {
+				log.WithFields(logFields).
+					Error("Could not delete image tag")
+			}
 		} else {
-			log.WithError(err).WithField("imageTag", inactiveTag).Error("Could not delete image")
+			log.WithFields(logFields).
+				Info("Would delete image tag")
```

Comment:
This can be even more reduced to:
```go
logEvent := log.WithFields(log.Fields{
    ....
})
logEvent.Info("msg")
```
It would still work with `logEvent.WithError(err).Error(...)` or additional fields. Saves you duplicated `.WithFields(logFields)`.